integration-tests/tests/e2e.rs

```diff
@@ -128,6 +128,10 @@ where
                         "encrypted_disk_label",
                         &encrypted_disk_loc.to_string_lossy(),
                     );
+                    cfg.insert(
+                        "user",
+                        users::get_effective_username().unwrap().to_str().unwrap(),
+                    );
```

Comment:
this just add the user, but is it checked anywhere? just making sure that the user __is indeed__ into the system

Reply:
creating user requires privileges, the way the test works is using a user(process owner) which is already present, so if you try with any random user name, the e2e test will fail. its kind of hack-y, so I lean on testing creation of new user manually or some complex automation of setting up the an VM and do the actual onboarding.

Reply:
The fdo client app runs as a privileged user so the tests should run as such no? The whole point of CI is test things automatically so that regressions from other PRs to this feature are appropriately picked up and we don't regress.

Reply:
Seems the test does run as superuser , changed the approach of the e2e test to actually do the onboarding instead of considering it as done.